### ISSUE TITLE
[UI/UX] Search option is not fully colored (some missing pixels at the top and from the right border)

### ISSUE THREAD

**Issue by mariano-aguero** [gnosis/conditional-tokens-explorer#600]
See comment https://github.com/gnosis/conditional-tokens-explorer/pull/571#issuecomment-723940788
Point 7

![98675299-212cf780-236b-11eb-940e-05359b82eee1](https://user-images.githubusercontent.com/1144028/98731477-8c32fa00-237c-11eb-83b8-a3673b26ae92.jpg)

**Comment by gabitoesmiapodo** [comment]
@elena-zh @mariano-aguero I could never reproduce the issue, but it could've been fixed here https://github.com/gnosis/conditional-tokens-explorer/pull/582#issuecomment-724909241

**Comment by elena-zh** [comment]
@gabitoesmiapodo , unfortunately, the issue is still reproducible for me. However, the issue us relevant for a 'certain' screen resolution: Size: 1536x712 (Landscape), Ratio: 192:89 (Browser - Chrome Version 86.0.4240.183), scale = 100%
![chrome](https://user-images.githubusercontent.com/70885163/98805881-5730b100-2429-11eb-9df7-ff44f800c1e5.jpg)

when a screen resolution is 1920x927 (Landscape), Ratio: 640:309, scale in [90-120%], everything looks good

**Comment by gabitoesmiapodo** [comment]
@elena-zh I think that's scaling the screen to force the viewport's size. If you're changing screen size through some type of emulation (say: Chrome extension, Chrome dev tools, etc.) some glitches are bound to happen (for example, drag and drop on Split Position completely fails when using Chrome Inspect, even if zoom is 100%).

If the component looks good when using the natural screen size on your computer and no extensions or tools, I don't think there's a problem (but if it persists, I'll have to dig more...).

The "fix" I implemented can be found here https://pr597--conditionaltokensexplorer.review.gnosisdev.com/

**Comment by elena-zh** [comment]
@gabitoesmiapodo , no, I'm not changing screen size through some type of emulation. The described above screen resolution details are my natural ones. 
The issue is still reproducible on my laptop  in https://pr597--conditionaltokensexplorer.review.gnosisdev.com/

**Comment by gabitoesmiapodo** [comment]
@elena-zh I put some more time into this, but I couldn't reproduce the issue.

I'm emulating your screen size (1536x712) in the screenshot below, but everything seems to be OK (tested on https://pr598--conditionaltokensexplorer.review.gnosisdev.com/#/merge and https://pr597--conditionaltokensexplorer.review.gnosisdev.com/#/merge)

![Screen Shot 2020-11-11 at 11 41 44](https://user-images.githubusercontent.com/4015436/98825581-6a388680-2413-11eb-806d-16cf8fbcf050.png)

Question: Is that black bar on top of your screen part of the browser, or is it some kind of plugin / extension? Could you test this when disabling that? Have you checked that browser zooming is not being used? (that works along dev tools' responsive zooming, but it could be the culprit), check if CTRL + 0 or CTRL + (+) or CTRL + (-) changes / fixes something.

Check this, please (I'm sorry if you already know or checked this, but I had this problem several times in the past, and it's a sneaky one to find or notice).

![Screen_Shot_2020-11-11_at_11_50_11](https://user-images.githubusercontent.com/4015436/98826489-84269900-2414-11eb-9a04-78dcb340a162.png)

I'm asking all these questions because these kind of issues (and the issues mentioned on https://github.com/gnosis/conditional-tokens-explorer/issues/599) are usually related to scaling and zooming. For example, the only way I could somewhat reproduce the issue was by zooming in a few times:

![Screen Shot 2020-11-11 at 11 43 17](https://user-images.githubusercontent.com/4015436/98826594-a7514880-2414-11eb-99c9-ced324284c9e.png)

As a last resort, please check if the issue(s) persist on a private browser tab, with all the extensions disabled and with a maximized browser window.

**Comment by elena-zh** [comment]
Hi @gabitoesmiapodo! Thanks a lot for your descriptive comment!

I have rechecked twice all the settings/scaling on my PC. The issue with checkboxes is still there even in the incognito mode
![incognito](https://user-images.githubusercontent.com/70885163/98925364-f2d42700-24e6-11eb-90c5-77fd13bdec70.jpg)

And the checkboxes start to look good only when the screen scale is 125%
![image](https://user-images.githubusercontent.com/70885163/98925481-16976d00-24e7-11eb-9082-2c6fd3ad7d74.png)

> Question: Is that black bar on top of your screen part of the browser, or is it some kind of plugin / extension? 

Yes, it is an extension. I switched in on only in order to show you my screen resolution. However, I test the app when the extension is switched off. 

Also, I have another screen with  1920x927 resolution. And the checkboxes look good there with scale = 100%. 

So, it appeared to be the device-specific issue then.. Only for my laptop... And if checkboxes look good now on other screen resolutions, we can close the issue...

Worth mentioning, that checkboxes in Metamask are also look not center-aligned in my laptop
![image](https://user-images.githubusercontent.com/70885163/98926005-cec51580-24e7-11eb-91c1-a6bdb00e6e8d.png)
